File: MODUL-6/Bahasa C/PRAK602-2310817120014-ALIYARAFFANAURAAYU.c
#include <stdio.h>

int main (void)
{
    int Kolom, Isi, N;
    scanf ("%d", &Kolom) ;
    int angka[Kolom] ;
    for (N = 0; N < Kolom; N++) {
        scanf ("%d", &Isi);
        angka[N] = Isi ;
    }
    for (N = 0; N < Kolom; N++) {
        printf ("\n%d ", angka[N] * (N + 1)) ;
    }
}

File: MODUL-6/Bahasa Python/PRAK602-2310817120014-ALIYARAFFANAURAAYU.py
Kolom = int(input())
Isi = list(map(int, input().split()))
angka = []
for N in range(0, Kolom) :
    angka.append(Isi[N])
print(" ")
for N in range(0, Kolom) :
    print(angka[N] * (N + 1), end=' ')